sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/DiagnosticProperty.cs

```diff
@@ -75,6 +75,11 @@ internal static class DiagnosticProperty
         /// </summary>
         public static string DeferActivityName { get; } = "ServiceBusReceiver.Defer";
 
+        /// <summary>
+        ///   The activity name associated with the delete operation.
+        /// </summary>
+        public static string DeleteActivityName { get; } = "ServiceBusReceiver.Delete";
```

Comment:
Should purge have one too?

Reply:
I don't think so, since it is just calling delete in a loop.   The service operation is "delete"

Reply:
Ok sg - that makes sense.

Reply:
I checked with @lmolkova on this and the rule of thumb is to instrument the convenience methods. The inner Delete spans will be automatically suppressed when we add an outer Purge span.

Reply:
okie dokie, I'll add one.

Reply:
FWIW we don't follow this pattern in our IAsyncEnumerable receive, so we should probably update that. I'll create an issue.

Reply:
Activity scope and dedicated ETW logs added for Purge.